Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM registry.access.redhat.com/ubi8/python-38:1-74
 USER 0
 RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
     yum remove -y npm && \
+    yum remove -y vim-minimal && \
```

Comment:
you dont need two lines here. you can combine it with previous line
`yum remove -y npm vim-minimal`